### PR TITLE
Bugfix: 1238833

### PR DESCRIPTION
**Pull request by skhiat**
Owner changed: @RSlysz 

Bug:
https://fogbugz.unity3d.com/f/cases/1238833/
To repro: enable guizmo (check the video).

Code:
Apparently in this context it's forbidden to use multicontext (message-error).

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
@skhiat please add a changelog (add the entry at the end of the Fixed section (not at the beginning :) )

**Comment by RSlysz** [comment]
Ho boy. This is a new things introduced in recent trunk it seams.
I dunno why it is not allowed anymore.
Note that we use multi-target for 
- FrameSettings (but in OnImspectorGUI)
- ReflectionProbes

@remi-chapelain have you tested with multiedition? Multi-target is used to compound operation on each target of the selection generally. So this can broke the multi-edition at some extend. This need a thorough check.

File: com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalProjectorEditor.cs [hunk 1]
```diff
@@ -150,22 +150,15 @@ public Bounds OnGetFrameBounds()
         public void UpdateMaterialEditor()
         {
             int validMaterialsCount = 0;
-            for (int index = 0; index < targets.Length; ++index)
```
Comment (RSlysz) [inline com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalProjectorEditor.cs:153]: So this problematic loop on target. 

It has no meanings on this file l.112 as it is used as a hook on each DecalProjector when the material change. So there we should not use the target notion. We should have a DecalProjector parameter to gather changes.

On line 109 though, the same function is called from OnEnable where multiselection can work on several target at a time.

It seams that this function (original version) is OK for line 109 but we should add another dedicated callback for the individual updated used in event

Comment (skhiat) [reply com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalProjectorEditor.cs:153]: The problem seems to come from another direction.

**Comment by RSlysz** [comment]
Replaced by PR https://github.com/Unity-Technologies/Graphics/pull/413

**Pull request closed without merging**